packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -107,22 +114,34 @@ class SnapshotInstanceItemPane extends StatelessWidget {
 
 /// `\v` adds vertical space
 const _snapshotDocumentation = '''
-Take a **heap snapshot** to view current memory allocation:
+1. Understand [Dart memory concepts](https://docs.flutter.dev/development/tools/devtools/memory#basic-memory-concepts).
+
+\v
+
+2. Take a **heap snapshot** to view current memory allocation:
+
+    a. In the Snapshots panel, click the ● button
+
+    b. Use the **Filter** button to refine the results
```

Comment:
Maybe we should add an "optional" tag to this instruction: `b. [Optional] use the **Filter** button to refine the results`